plugins/wls.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+    name: wls
+spec:
+    version: v1.0.0
+    homepage: 'https://github.com/mohatb/kubectl-wls'
+    shortDescription: 'Windows-linux-shell allows you to start a shell on windows or linux nodes'
```

Comment:
I don't understand anything from this sentence.

Reply:
> wls: stands for windows-linux-shell 
> it allows the user to get into a windows or linux shell through this plugin, for linux it uses nsenter, for windows, it creates a pod on a linux node and then ssh to a windows node.

Reply:
can you test the yaml and the plugin again ?